applications/crossbar/src/modules/cb_webhooks.erl

```diff
@@ -86,12 +86,20 @@ maybe_revise_schema(MasterDb, SchemaJObj) ->
         {'error', _E} ->
             lager:warning("failed to find registered webhooks: ~p", [_E]);
         {'ok', Hooks} ->
-            revise_schema(SchemaJObj, [kz_json:get_value(<<"key">>, Hook) || Hook <- Hooks])
+            ToRemovHooks = [<<"callflow">>
```

Comment:
Not a big deal, but the variable should be `ToRemoveHooks` (missing the second e)